README.md

```diff
@@ -1,4 +1,7 @@
 # Data Exporter
+A cli tool which allows exporting and importing of user-data (settings,files and shares) from one instance in to another.
```

Comment:
Proposal:
``settings,files and shares`` —> ``like settings,files and shares``
``from one instance`` —> ``to migrate data from one instance``